contracts/HubPool.sol

```diff
@@ -216,23 +236,20 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         bool sendEth
     ) public nonReentrant {
         // Can only send eth on withdrawing liquidity iff this is the WETH pool.
-        require(l1Token == address(l1Weth) || !sendEth, "Cant send eth");
-        uint256 l1TokensToReturn = (lpTokenAmount * _exchangeRateCurrent()) / 1e18;
-
-        // Check that there is enough liquid reserves to withdraw the requested amount.
-        // require(liquidReserves >= (pendingReserves + l1TokensToReturn), "Utilization too high to remove"); // TODO: add this when we have liquid reserves variable implemented.
+        require(pooledTokens[l1Token].isWeth || !sendEth, "Cant send eth");
+        uint256 l1TokensToReturn = (lpTokenAmount * _exchangeRateCurrent(l1Token)) / 1e18;
```

Comment:
Can `_exchangeRateCurrent` be 0?

Reply:
nope. it should never be 0. it should start at 1 and grow over time as each LP token is redeemable for more underlying.